### PR TITLE
doc: Improve Information about spec

### PR DESCRIPTION
**Pull request by bbernays**
#### Summary

Add information about `spec` field

### CODE REVIEW

File: website/pages/docs/reference/destination-spec.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Specifies the update method to use when inserting rows. The exact semantics depe
 - `overwrite-delete-stale`: Same as `overwrite`, but stale rows are also deleted.
 - `append`: Rows are never overwritten or deleted, only appended.
 
-## Development
+### spec
 
-Spec is defined in [plugin-sdk](https://github.com/cloudquery/plugin-sdk/blob/main/specs/destination.go#L12)
+(`object`, optional)
+
+Plugin specific configurations. See [plugin](..plugins/destinations) specific documentation for more information.
```
Comment (erezrokah) [inline website/pages/docs/reference/destination-spec.md:50]:
```suggestion
Plugin specific configurations. Visit [destination plugins](docs/plugins/destinations) documentation for more information.
```

File: website/pages/docs/reference/source-spec.md [hunk 2]
```diff
@@ -63,3 +63,9 @@ Sets a global limit on the number of tables to sync concurrently.
 
 Sets an (approximate) global limit on the number of concurrent requests performed to fetch further details about resources. Note that this limit only applies to top-level tables and not their child relations.
 
+### spec
+
+(`object`, optional)
+
+Plugin specific configurations. See [plugin](..plugins/sources) specific documentation for more information.
```
Comment (erezrokah) [inline website/pages/docs/reference/source-spec.md:70]:
```suggestion
Plugin specific configurations. Visit [source plugins](docs/plugins/sources) documentation for more information.
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
The links are broken I think, visit https://cloudquery-e8brotdk5-cloudquery.vercel.app/docs/reference/destination-spec#spec-1 to test.

I could go either way with `visit` or `see`

**Review by cqgaurav — APPROVED** [review]

**Comment by cqgaurav** [comment]
Looks good, please address Erez's comments :)

**Review by erezrokah — APPROVED** [review]

File: website/pages/docs/reference/destination-spec.md [hunk 3]
```diff
@@ -43,6 +43,8 @@ Specifies the update method to use when inserting rows. The exact semantics depe
 - `overwrite-delete-stale`: Same as `overwrite`, but stale rows are also deleted.
 - `append`: Rows are never overwritten or deleted, only appended.
 
-## Development
+### spec
 
-Spec is defined in [plugin-sdk](https://github.com/cloudquery/plugin-sdk/blob/main/specs/destination.go#L12)
+(`object`, optional)
+
+Plugin specific configurations. Visit [destination plugins](docs/plugins/destinations) documentation for more information.
```
Comment (erezrokah) [inline website/pages/docs/reference/destination-spec.md:50]:
```suggestion
Plugin specific configurations. Visit [destination plugins](/docs/plugins/destinations) documentation for more information.
```

File: website/pages/docs/reference/source-spec.md [hunk 4]
```diff
@@ -63,3 +63,9 @@ Sets a global limit on the number of tables to sync concurrently.
 
 Sets an (approximate) global limit on the number of concurrent requests performed to fetch further details about resources. Note that this limit only applies to top-level tables and not their child relations.
 
+### spec
+
+(`object`, optional)
+
+Plugin specific configurations. Visit [source plugins](docs/plugins/sources) documentation for more information.
```
Comment (erezrokah) [inline website/pages/docs/reference/source-spec.md:70]:
```suggestion
Plugin specific configurations. Visit [source plugins](/docs/plugins/sources) documentation for more information.
```

**Comment by erezrokah** [comment]
Added another fix as the links should start with `/`

**Pull request merged**